scripts/precook-node-modules.js

```diff
@@ -23,5 +23,5 @@ fs.ensureDir('tmp')
     symlinkDirectory(root, path.join(precooked, 'ember-cli-fastboot'));
   })
   .catch(function(e) {
-    console.log(e);
+    console.log(e); // eslint-disable-line no-console
```

Comment:
Agreed. Basically everybody starting to work on this repo some time stumbles upon this. Same for me 😆